docs/en/cloud/marketplace.md

```diff
@@ -4,20 +4,21 @@ slug: /en/cloud/marketplace
 title: CSP Marketplace
 ---
 
-## How can I start using ClickHouse Cloud connected to my AWS/GCP/Azure account billing?
+## Sign up for ClickHouse Cloud connected to your Cloud Service Provider (CSP) - AWS/GCP/Azure - billing account
 
 ### AWS 
 
-- Log into the AWS console using your AWS account
-- Navigate to the [ClickHouse Cloud at aws marketplace](https://aws.amazon.com/marketplace/pp/prodview-jettukeanwrfc)
+- Log into the AWS console using your AWS account. This is the account you intend to subscribe to the ClickHouse Cloud service.
+- Navigate to [ClickHouse Cloud on AWS marketplace](https://aws.amazon.com/marketplace/pp/prodview-jettukeanwrfc) product listing
 - Click "View purchase options"
 - In the "Contract Options" section of the page, enter any number in the Units field. This will not affect the price your pay as the price for these units for the public offering is $0. These units are usually used when accepting a private offer from ClickHouse Cloud.
```

Comment:
```suggestion
1. Click **View purchase options**.
1. In the **Contract Options** section of the page, enter any number in the **Units** field. This will not affect the price your pay as the price for these units for the public offering is $0. These units are usually used when accepting a private offer from ClickHouse Cloud.
```